packages/concerto-metamodel/lib/metamodel.cto

```diff
@@ -30,6 +30,7 @@ concept Range {
 concept TypeIdentifier {
   o String name
   o String namespace optional
+  o TypeIdentifier[] typeParameters optional
```

Comment:
How do typeParameters (optionally) contribute to type identity?

Reply:
Ah, this is unused. An artifact from an earlier iteration
```suggestion
```